### DISCUSSION TITLE
The ability to delete your supabase account

### DISCUSSION THREAD

**Discussion by Isaiah-Hamilton** [supabase/supabase#5036] [Feature Requests] [answered]
Let's say that someone signed up for supabase but for some reason didn't like it and wanted to delete their account. Currently, there is no way (That I know of) for someone to delete their supabase account. I think there should be an option for people to delete their account in the supabase dashboard.

**Comment by kk21** [comment]


**Reply by silentworks** [reply]
This is not what they original message is asking. He is asking how to delete his account that he signed up to Supabase with from Supabase itself, not a user from one of his Supabase projects.

**Comment by silentworks** [comment]
Good suggestion.

**Comment by BrunoBernardino** [comment]
I find it surprising there isn't a way to do this. Sounds like a GDPR nightmare waiting to happen.

**Comment by 1mansh0w** [comment]
So, how do we delete our supabase account? There should be an option for it

**Comment by Olyno** [comment]
I would like to edit the suggestion, and changing it to "The ability to update your supabase account" in general. We can't update our account to add/remove information, and so we can't delete it.

**Comment by harryfoster** [comment]
Has this been added yet? I got an email about my projects being suspended and I just want to remove the account completely?

**Reply by silentworks** [reply]
You can send an email to support to request your account removal. https://app.supabase.io/support/new

**Reply by talamaska** [reply]
Why not just add the delete my account button, as it is required from GDPR.

**Comment by inian** [comment, marked as answer]
You can request account deletion [here](https://supabase.com/dashboard/account/me)
